src/ucp/wireup/address.c

```diff
@@ -1040,7 +1042,8 @@ ucs_status_t ucp_address_unpack(ucp_worker_t *worker, const void *buffer,
                 ucs_trace("unpack addr[%d] : eps %u md_flags 0x%"PRIx64
                           " tl_iface_flags 0x%"PRIx64" tl_event_flags 0x%"PRIx64
                           " bw %e + %e/n ovh %e lat_ovh %e dev_priority %d a32 "
-                          "0x%lx/0x%lx a64 0x%lx/0x%lx",
+                          "0x%"PRIx64"/0x%"PRIx64 
+                          " a64 0x%"PRIx64"/0x%"PRIx64,
```

Comment:
wrap lines

Reply:
Could you check again? I wrapped by 80 characters.